types/ckeditor__ckeditor5-link/src/utils/manualdecorator.d.ts

```diff
@@ -0,0 +1,17 @@
+import { BindChain, Observable } from '@ckeditor/ckeditor5-utils/src/observablemixin';
+
+export default class ManualDecorator implements Observable {
+    readonly attributes: Record<string, string>;
+    readonly defaultValue: boolean;
+    readonly id: string;
+    readonly label: string;
+    value: boolean;
+
+    constructor(args: { id: string; label: string; attributes: Record<string, string>; defaultValue?: boolean });
```

Comment:
```suggestion
    constructor(args: { id: string; label: string; attributes: Record<string, string>; defaultValue?: boolean });
```
I'd avoid this:
- it uses differen param names (should read `config`)
- it uses inline type, which cannot be reused
https://ckeditor.com/docs/ckeditor5/latest/api/module_link_utils-ManualDecorator.html

Reply:
>     * it uses differen param name (should read `config`)

Fixed this.

>     * it uses inline type, which cannot be reused
>       https://ckeditor.com/docs/ckeditor5/latest/api/module_link_utils-ManualDecorator.html

But the type for this constructor is not formally defined and/or exported. That's why I chose an inline type.
If somebody needs to get these parameters, they can use `ConstructorParameters<T>`.